clients/python/examples/rest.py

```diff
@@ -0,0 +1,16 @@
+from raccoon_client.client import Event
+from raccoon_client.rest.client import RestClient
+from raccoon_client.rest.option import RestClientConfigBuilder
+from raccoon_client.serde.enum import Serialiser, WireType
+
+if __name__ == '__main__':
+    event_data = {"a": "field a", "b": "field b"}
+
+    config = RestClientConfigBuilder().with_url("http://localhost:8080/api/v1/events")\
+        .with_serialiser(Serialiser.PROTOBUF)\
```

Comment:
Correct. I guess I missed converting all fields when converting the example. Ideally, the serialiser should be JSON here and not protobuf at all. This will give error saying it's not serialisable, due to incorrect typing.

Reply:
Fixed and added multple examples

Reply:
Can you resolve this post checking.